module/VuFind/src/VuFind/Search/Base/Options.php

```diff
@@ -741,19 +743,19 @@ public function getHiddenSortOptions()
      *
      * @return array
      */
-    public function getFacetSortOptions($facet = '*')
+    public function getFacetSortOptions(string $facet = '*'): array
     {
         return $this->facetSortOptions[$facet] ?? $this->facetSortOptions['*'] ?? [];
     }
 
     /**
      * Get the default sort option for the specified search handler.
      *
-     * @param string $handler Search handler being used
+     * @param ?string $handler Search handler being used
```

Comment:
We should probably specify what null does:

```suggestion
     * @param ?string $handler Search handler being used (null for default)
```